package.json

```diff
@@ -53,7 +53,8 @@
     "dev-reinit-windows": "cd packages/server && npm run dev-reinit-db-windows",
     "dev-nginx": "",
     "cleanup-tests": "node tests/cleanup-tests.js",
-    "test": "npm run test:projects && npm run test:packages",
+    "pretest": "cross-env TEST=true npm run dev-reinit",
+    "test": "cross-env TEST=true npm run test:projects && cross-env TEST=true npm run test:packages",
```

Comment:
lets move these cross env to test:packages and test:projects so they can be run separately on the test database

Reply:
Done